### ISSUE TITLE
Google won't let user sign in because reported Chromium version is too old

### ISSUE THREAD

**Issue by cdavis5e** [brave/brave-browser#40724]
### Description

When signing into Google, after entering the user's email address, the site claims it couldn't sign the user in and suggests that it is because the browser is insecure and/or unsupported.

The only thing I noticed that could be causing this is that the user agent string claims to be an antediluvian version of Chrome (88). This is the UA string in question:

Mozilla/5.0 (Macintosh; Intel Mac OS X 11_3_0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/88.0.4324.192 Safari/537.36

### Steps to reproduce

1. Navigate to a Google web site.
2. Click "sign in."
3. Enter your Google account email address.

### Actual result

Site claims it "couldn't sign you in" because "this browser or app may not be secure."

https://github.com/user-attachments/assets/93208e79-6155-4b72-9fd3-511122b62340

### Expected result

Site should've prompted for password and/or 2FA data.

### Reproduces how often

Easily reproduced

### Brave version (brave://version info)

Brave | 1.69.153 Chromium: 128.0.6613.85 (Official Build) (x86_64)
Revision | a69e96756922f88cbce3482971d70cfd1ee6f872
OS | macOS Version 12.7.3 (Build 21H1006)

### Channel information

- [X] release (stable)
- [ ] beta
- [ ] nightly

### Reproducibility

- [X] with Brave Shields disabled
- [ ] with Brave Rewards disabled
- [ ] in the latest version of Chrome

### Miscellaneous information

Disabling Brave shields didn't help.
Nor did disabling uMatrix.
I'm 99.9999% certain it's the UA string. Is there a way to make Brave use a different one showing a more recent Chromium version?

**Comment by RPGhugo** [comment]
Has this issue been resolved or is it still ongoing?

**Comment by cdavis5e** [comment]
Yes, it is still ongoing. (I apologize for taking so long to get back to you.)

For some reason, I only see this problem on my MacBook Pro running Monterey. On my Mac Pro running Sequoia, it works just fine. I guess only about one in 100000 at most would see this problem, given its nature. I assume you're randomizing aspects of the UA string to foil fingerprinting, right?
